src/features/user/TreeMapper/Import/components/SpeciesAutoComplete.tsx

```diff
@@ -14,14 +14,27 @@ import { ErrorHandlingContext } from '../../../../common/Layout/ErrorHandlingCon
 
 const config = tenantConfig();
 
-export default function SpeciesSelect(props: {
+interface Props<
+  TFieldValues extends FieldValues,
+  TName extends FieldPath<TFieldValues>
+> {
   label: React.ReactNode;
-  name: string;
-  defaultValue?: String | undefined;
+  name: TName;
   width?: string | undefined;
-  control: any;
-  mySpecies?: any;
-}) {
+  control: Control<TFieldValues>;
+  // mySpecies?: any;
+}
+
+export default function SpeciesSelect<
+  TFieldValues extends FieldValues,
+  TName extends FieldPath<TFieldValues>
+>({
+  label,
+  name,
+  width,
+  // mySpecies,
```

Comment:
remove the commented part , If it not require

Reply:
Leaving the comments in case this is needed for future reference. Will remove later if it is clear that this will not be needed.